### ISSUE TITLE
Bag medallions bag is bugget

### ISSUE THREAD

**Issue by Evilchosen** [WarEmu/WarBugs#19650]
![image](https://user-images.githubusercontent.com/82473452/144708704-c3f75fe4-605c-4df0-aa15-19c3a622c6fd.png)

Missing all my invader and royals

**Comment by fbodin** [comment]
I have the same problem +1

Char on Order side: Fobzee

**Comment by Lheana** [comment]
Have you checked in your bank in KaK/K8P to see if your missing currencies are there?

**Comment by xpander69** [comment]
as of patch 4.12.2021

```
- Altdorf and Karaz-A-Karak, as well as The Inevitable City and Karak Eight Peaks no longer share a banks. All items in the bank are now exclusivly available in the Empire and Chaos City banks.

- The additional currency bag has been removed and all items have been moved to the Dwarf and Greenskin City banks.
```
check your banks in the dwarf/greenskin cities

**Comment by fbodin** [comment]
Thx, the missing currencies has been found in Karaz-A-Karak

**Comment by lilimx** [comment]
@Evilchosen can you confirm that you found your medallions?

**Comment by lilimx** [comment]
Hey,

I am closing this ticket now as we have not received any further information regarding the problem.
The issue was fixed on 4.12.2021 and we haven't received any further bug reports.
If you are still the opinion that the issue still persits, please open a new ticket with proper evidences.
